frontend/assets/translations/fr.json

```diff
@@ -1,921 +1,918 @@
+    "email": "E-mail",
+    "receiveUpdatesFromToolJet": "Vous recevrez des mises à jour de l'équipe ToolJet (1-2 e-mails chaque mois, nous ne spammons pas)",
+    "finishSetup": "Configuration de la fin",
+    "skip": "Sauter"
+  },
+  "redirectSso": {
+    "upgradingTov1.13.0": "Mise à niveau vers V1.13.0 et supérieur.",
+    "fromV1.13.0": "De V1.13.0, nous avons introduit",
+    "multiWorkspace": "Espace multi-travaux",
+    "singleSignOnConfig": "Les configurations liées à la connexion unique sont déplacées des variables d'environnement vers la base de données. Veuillez référer ceci",
+    "link": "Lien",
+    "toConfigureSSO": "Pour configurer SSO.",
+    "haveGoogleGithubSSo": "Si vous avez des configurations SSO Google ou GitHub avant la mise à niveau et l'espace multi-works désactivé, les configurations de Thesso seront migrées pendant la mise à niveau, mais vous devez reconfigurer l'URL de redirection du côté fournisseur SSO. Les URL de redirection pour chaque SSO sont données ci-dessous.",
+    "isMultiWorkspaceEnabled": "Si vous avez activé Multi-Workspace, les configurations SSO ne seront pas migrées lors de la mise à niveau, vous devez donc reconfigurer le SSO sous l'espace de travail respectif.",
+    "youHaveEnabled": "Vous avez activé",
+    "setupSsoWorkspace": "Veuillez vous connecter avec le mot de passe et vous pouvez configurer SSO en utilisant Workspace",
+    "manageSsoMenu": "Gérer le menu SSO.",
+    "youHaveDisabled": "Vous avez handicapé",
+    "configureRedirectUrl": "Veuillez configurer l'URL de redirection du côté fournisseur SSO.",
+    "google": "Google",
+    "redirectUrl": "URL de redirection:",
+    "gitHub": "Github"
+  },
+  "oAuth2": {
+    "pleaseWait": "S'il vous plaît, attendez...",
+    "authSuccess": "Auth réussi, vous pouvez fermer cet onglet maintenant.",
+    "authFailed": "L'auteur a échoué"
+  },
+  "widgetManager": {
+    "commonlyUsed": "fréquemment utilisé",
+    "layouts": "disposition",
+    "forms": "formes",
+    "intregrations": "intégrations",
+    "others": "les autres",
+    "noResults": "Aucun résultat trouvé",
+    "tryAdjustingFilterMessage": "Essayez d'ajuster votre recherche ou votre filtre pour trouver ce que vous recherchez.",
+    "clearQuery": "requête claire"
+  },
+  "widget": {
+    "common": {
+      "properties": "Propriétés",
+      "events": "Événements",
+      "layout": "Disposition",
+      "styles": "modes",
+      "general": "Générale",
+      "validation": "Validation",
+      "documentation": "{{componentMeta}} documentation",
+      "widgetNameEmptyError": "Le nom du widget ne peut pas être vide",
+      "componentNameExistsError": "Le nom du composant existe déjà",
+      "invalidWidgetName": "Nom du widget non valide. Devrait être unique et n'inclure que des lettres, des chiffres et un soulignement."
+    },
+    "commonProperties": {
+      "visibility": "Visibilité",
+      "disable": "Désactiver",
+      "borderRadius": "Rayon de frontière",
```

Comment:
😵‍💫 
```suggestion
      "borderRadius": "Rayon de la bordure",
```

Reply:
@roiLeo , Can you please review the changes that I have made as you instructed, Thanks :)